FBC/DebtAndEquities/CreditEvents.rdf

```diff
@@ -221,30 +227,24 @@
 	</owl:Class>
 	
 	<owl:ObjectProperty rdf:about="&fibo-fbc-dae-cre;hasDefaultThresholdAmount">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-acc-cur;hasMonetaryAmount"/>
 		<rdfs:label xml:lang="en">has default threshold amount</rdfs:label>
 		<rdfs:domain rdf:resource="&fibo-fbc-dae-cre;EntitySpecificCreditEvent"/>
```

Comment:
No, the threshold should be a property of the contract - that is used to determine whether a DefaultEvent gets triggered. It does not make sense for it to be a property of a credit event that has not occurred